atd-vze/src/views/Crashes/Maps/CrashMap.js

```diff
@@ -92,8 +92,6 @@ export default class CrashMap extends Component {
         <Marker
           latitude={this.props.data.latitude_primary}
           longitude={this.props.data.longitude_primary}
-          offsetLeft={-20}
-          offsetTop={-10}
```

Comment:
🤣  too simple, haha

Reply:
Yep! 😆 I'm glad that it ended up being simpler than I thought.